### ISSUE TITLE
ffmpeg do not support fontconfig

### ISSUE THREAD

**Issue by catcheroftime** [conan-io/conan-center-index#22578]
### What is your question?

I want to use ffmpeg's drawtext to add text, but the library provided by conan doesn't add the --enable-libfontconfig compilation parameter

**Comment by uilianries** [comment]
@catcheroftime Hello! I see you're interested in a new feature into FFmpeg. Would you be open to submitting a pull request with this feature? If you need any assistance or have questions about the process, please don't hesitate to reach out. The Conan community is here to help!

**Comment by cubisttriangle** [comment]
@uilianries can this be closed now?

**Comment by jcar87** [comment]
Yup, closed by https://github.com/conan-io/conan-center-index/pull/27379

Thanks @cubisttriangle!
